src/Doctrine/MigrationFactoryDecorator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Wallabag\Doctrine;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\Migrations\Version\MigrationFactory;
+
+class MigrationFactoryDecorator implements MigrationFactory
```

Comment:
Can you put a description for that class?

Reply:
@j0k3r done!